Src/PerformanceCollector/Unit.Tests/QuickPulse/QuickPulseTelemetryProcessorTests.cs

```diff
@@ -402,22 +404,28 @@ public void QuickPulseTelemetryProcessorCollectsFullTelemetryItems()
             // ACT
             var request = new RequestTelemetry()
                               {
-                                  Id = Guid.NewGuid().ToString(),
+                                  Name = Guid.NewGuid().ToString(),
```

Comment:
I'm trying to understand, why is the new Guid set to an "Id" property in the others but "Name" here, particularly when this used to be Id?